<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QDialog" name="main_window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>462</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>468</width>
    <height>462</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>468</width>
    <height>462</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>468</width>
    <height>462</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Decrypt Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(160, 160, 160)</string>
  </property>
  <widget class="QLabel" name="l_header1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Data Decryption</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_path">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_path">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Path to folder or file</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_todo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pb_progress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>431</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_decrypt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Decrypt Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_pass">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password to decrypt files</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_password">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
